packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -712,9 +712,10 @@ class ShortcutManager extends ChangeNotifier with Diagnosticable {
         final Action<Intent>? action = Actions.maybeFind<Intent>(
           primaryContext,
           intent: matchedIntent,
+          createsDependency: false,
         );
         if (action != null && action.isEnabled(matchedIntent)) {
-          Actions.of(primaryContext).invokeAction(action, matchedIntent, primaryContext);
+          Actions.invoke(primaryContext, matchedIntent);
```

Comment:
This is unfortunate that we have to do the "find action" work twice here since we also have to find the correct dispatcher. @gspencergoog do you think it will be better to introduce a `tryInvoke` API that also tells the caller whether a suitable `Action` was found?

Reply:
Yeah, that seems reasonable.